### ISSUE TITLE
I cannot perform playback controls or system commands

### ISSUE THREAD

**Issue by Angenios** [morrolinux/mpradio-py#57]
![immagine](https://user-images.githubusercontent.com/37461401/82117202-fae91f00-976e-11ea-8585-0706333e9c45.png)

It denies me the permission to perform such commands even if I use `sudo`.

**Comment by morrolinux** [comment]
Yeah in GNU/Linux sudo doesn't survive the > redirection operator. You should either use `sudo -i` to become root or use tee: `echo pause | sudo tee /tmp/mpradio_bt `

**Comment by Angenios** [comment]
> 
> 
> Yeah in GNU/Linux sudo doesn't survive the > redirection operator. You should either use `sudo -i` to become root or use tee: `echo pause | sudo tee /tmp/mpradio_bt `

It still doesn't reply to any command, and I've noticed another problem but I'll open a new issue about it in order to make order between problems.
